.env.production

```diff
@@ -8,6 +8,7 @@ MIXPANEL_TOKEN='b5c28d653684157f831d6b604d5e1956'
 ENABLE_TEST_IDP=YES
 GCM_SENDER_ID='317157111831'
 DEBUG_REMOTE_PUSH_NOTIFICATION=NO
+DEBUG_BIOMETRIC_IDENTIFICATION=NO
```

Comment:
Can we add a section on the README about this field? (eventually I'd like to document all of them)

Reply:
Added a table of availables env variables with only DEBUG_BIOMETRIC_IDENTIFICATION for now. We can add others later.